src/core/searchUtils.js

```diff
@@ -35,6 +41,14 @@ export function convertQueryParamsToFilters(params) {
   }, {});
 }
 
+export function convertOperatingSystemToFilterName(name) {
+  if (name in operatingSystems) {
+    return operatingSystems[name];
+  }
+
+  return '';
```

Comment:
should this be `undefined` so that the query string doesn't get a value?

Reply:
An empty value will also be removed... but yes, that would make more sense semantically.